src/index.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="styles/style.css" />
+    <title>otus-three-column-layout</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-wrapper">
+        <a href="#">Услуги</a>
+        <a href="#">Партнеры</a>
+        <a href="#">Оплата</a>
+      </div>
+    </header>
+    <main class="content">
+      <section class="left-section">
+        <h3>Полезные ресурсы</h3>
+        <ul class="useful-resources">
+          <li><a href="#">Показы</a></li>
+          <li><a href="#">Статистика продаж</a></li>
+          <li><a href="#">Интервью с руководителями</a></li>
+        </ul>
```

Comment:
Навигацию лучше "оборачивать" в nav
```suggestion
        <nav>
          <ul class="useful-resources">
            <li><a href="#">Показы</a></li>
            <li><a href="#">Статистика продаж</a></li>
            <li><a href="#">Интервью с руководителями</a></li>
          </ul>
        </nav>
```